Fuchas/Kernel/boot.lua

```diff
@@ -292,17 +281,17 @@ end, function(err)
 		gpu.setBackground(0x0000FF)
 		gpu.fill(1, 1, 160, 50, " ")
 		write([[A problem has been detected and Fuchas
-has shutdown to prevent damage
+has halted to prevent damage
 to your computer.
 
 Error trace:
 ]] .. err .. " \n \n " .. [[
 
 If this is the first time you've seen
-this BSOD screen, restart your
+this error screen, restart your
```

Comment:
i liked BSOD :( + it's blue and directly inspired on BSODs

Reply:
yeah, but it sounds off when read

Reply:
the windows error screen says something almost exactly like that anyway iirc

Reply:
i guess yea then